### PR TITLE
[TS-313] fixed dashboard bugs 313, 171 & 305

### PR DESCRIPTION
**Pull request by itsShaVawn**
Dashboard grid and component spacing is now optimal for different screen sizes.

### CODE REVIEW

**Comment by tungleu** [comment]
image
These 2 components are still overlapping from my end

**Comment by itsShaVawn** [comment]
> image These 2 components are still overlapping from my end

fixed

File: src/components/Portfolio/MonthlyProfitLoss.vue [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <template>
-  <v-card min-width="363" min-height="164" max-height="164" class="pa-3 white--text" color="#3F51B5">
-    <v-row justify="space-between" align="start" class="mb-6">
-      <v-col cols="8" class="text-left">
+  <v-card :min-width="$vuetify.breakpoint.smAndUp ? 250 : 350" min-height="168" class="pa-3 white--text d-flex flex-column" color="#3F51B5">
```
Comment (tungleu) [inline src/components/Portfolio/MonthlyProfitLoss.vue:2]: Nit: line is too long here

File: src/components/Portfolio/MonthlyProfitLoss.vue [hunk 2]
```diff
@@ -10,15 +10,14 @@
       </v-col>
       <!-- Top right Empty Icon -->
     </v-row>
-    <v-row align="center" justify="center">
+    <v-row align="center" no-gutters>
       <v-col>
         <v-icon class="float-left pr-1" size="20" color="white">{{percentChangeLogo()}}</v-icon>
         <v-list-item-title class="float-left pr-2 subtitle-2" data-cy='pl-percent-change'>{{ percentChange }}%</v-list-item-title>
```
Comment (tungleu) [inline src/components/Portfolio/MonthlyProfitLoss.vue:16]: Nit: line is too long here

**Review by tungleu — APPROVED** [review]
Just a few nits, overall good job on fixing those bugs! 👍

**Review by tungleu — APPROVED** [review]

**Review by Ming424 — APPROVED** [review]
looks good in my run, good job

**Pull request merged**